external-crates/move/crates/move-bytecode-source-map/src/source_map.rs

```diff
@@ -490,46 +490,38 @@ impl SourceMap {
 
     /// Create a 'dummy' source map for a compiled module or script. This is useful for e.g. disassembling
     /// with generated or real names depending upon if the source map is available or not.
-    pub fn dummy_from_view(view: &BinaryIndexedView, default_loc: Loc) -> Result<Self> {
-        let module_ident = match view {
-            BinaryIndexedView::Module(..) => {
-                let module_handle = view.module_handle_at(ModuleHandleIndex::new(0));
-                let module_name = ModuleName(Symbol::from(
-                    view.identifier_at(module_handle.name).as_str(),
-                ));
-                let address = *view.address_identifier_at(module_handle.address);
-                ModuleIdent::new(module_name, address)
-            }
+    pub fn dummy_from_view(module: &CompiledModule, default_loc: Loc) -> Result<Self> {
```

Comment:
`from_module`? don't know, whatever.... :)

Reply:
I might do a separate pass for these sorts of things. Trying to keep the diff clean as I've made one mistake already

Reply:
sounds like a good plan!